spring-kafka-docs/src/main/asciidoc/retrytopic.adoc

```diff
@@ -0,0 +1,449 @@
+[[retry-topic]]
+== Non-Blocking Retries
+
+Achieving non-blocking retry / DLT functionality with Kafka usually requires setting up extra topics and creating and configuring the corresponding listeners.
+Since 2.7 Spring Kafka offers the `@RetryableTopic` annotation and `RetryTopicConfiguration` class to simplify that bootstraping.
```

Comment:
```suggestion
Since 2.7 Spring for Apache Kafka offers the `@RetryableTopic` annotation and `RetryTopicConfiguration` class to simplify that bootstraping.
```

The foundation requires us to mention Apache in quotes like this; we can use `spring-kafka` when referring to the jar.

Reply:
Done